src/providers/implementations/AnthropicProvider.ts

```
@@ -0,0 +1,126 @@
+    };
+
+    if (apiKey.length < 20) {
+      return {
+        isValid: false,
+        error: 'Anthropic API key appears to be too short'
+      };
+    }
+
+    return {
+      isValid: true
+    };
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)